### ISSUE TITLE
Event Planning for 2022

### ISSUE THREAD

**Issue by GoldenBit0** [eea-oasis/baseline-outreach#17]
# Item

# Description 
The Outreach team - events work group is responsible for keeping track of events that Baseline related activities (speakers, presentations, developer activities) would be advantageous to the Baseline Protocol by sharing the value and/or growing the community. Since technical, business, and events are typically planned and finalized far in advance, the group needs an efficient way of organizing and managing what is coming up, applications, and community members who can represent the Baseline community.
[Events WG Drive](https://drive.google.com/drive/folders/1C6rdAJirw2Wzm5fkyuS-zVaMSq0eJc1P?usp=sharing)

# Definition of 'Done'

- [x] Event tracker format finalized
- [x] Major events and information for 2022 are documented
- [ ] Deadlines for applications are specified
- [ ] Applications for all applicable events are submitted

# Steps

- [x] Draft tracker template
- [x] Update/iterate template based on group feedback
- [ ] Apply to all events with open applications
- [ ] Determine community reps for events based on region/availability/interest 

# Notes
Once this activity is complete, the Outreach team will create collateral (presentation materials), determine what can be provided for distribution for events (stickers, merch, etc), and identify follow up action items. The group will also need to start identifying which events to look into for 2023.

**Comment by GoldenBit0** [comment]
2/14/22:

- Sheet will be updated and need to focus on which exact events we’d like to be at, and target as early as possible 
- Sheet will be added to the outreach drive
- Add list of people and locations who have interest in speaking at events
- Can start outlining topics and focuses for speakers (general presentation for speakers)

**Comment by GoldenBit0** [comment]
2/28/22:

**Tracker**
- Need to finish going through potential events tab and sifting out events we're interested in
- Those interested in representing add name to doc by event
- Group members can add additional events 

**Notes**
- Events WG needs to determine presence at events and resources needed
- DevConnect- Will have prioritization since it is coming up. Jane can work with Jack to determine next steps/needs for event logistics. Sonal will find out which orgs (ConsenSys, EEA, Oasis, Provide, others) are already participating at event 
- Oasis can work with sponsor orgs once list is narrowed down to determine fees
- Put together slide deck from prospective of our events WG to have resources for presenting 
- Add security, tech, enterprise, ZK, use case centric (fintech, supply chain, etc) events to the list

**Comment by GoldenBit0** [comment]
3/14/22:

**Tracker:**

- Have group take one more pass through event tracker

**Notes**

- Look at Consensus event, and others where web services are attending
- Work through DevConnect items quickly

**Comment by GoldenBit0** [comment]
3/28/22:

- Deadlines are added to the events sheet in 'cfp', and some have rolling dates
- **High priority this week-** Black Hat event, info sec event: submission deadline 4/4 (SP, JW, YB, MH, CT can assist Jane). Jane will attempt to get a list of representatives from different TC's. Sonal will connect with Eric B.
- Mesh team will look into Consensus event
- Sonal will update issue to remain open throughout the year

**Comment by GoldenBit0** [comment]
4/11/22:

- Black Hat submission complete, awaiting response. Oasis possibly able to have presence at event, Jane will stay in touch
- Sonal in contact with Consensus contact, will update team 
- FinTech DevCon Denver, paper is due on May 1, 
- Submissions from Oasis need to be vendor neutral about Baseline Protocol, can include demos from companies, companies can also submit own applications too: Marc Haddle taking lead for proposal

**Comment by GoldenBit0** [comment]
4/25/22:

- Consensus contact was not relevant to us, will keep an eye out for other corporate events occurring during that week.
- Marc will draft FinTech DevCon paper and send to chat mid-week, optionally Jane can submit on behalf of Oasis 
- DevCon Bogota gives us a good timeline to plan. Provide is in contact now, Mesh may attend, and others.
- Are there any events that EEA is going to be part of?

**Comment by GoldenBit0** [comment]
5/9/22:

- Marc's paper submitted for FinTech DevCon
- Marc working on a paper for event alongside BlackHat now, will send to group to assess how to focus piece for infosec 
- Marketing barter in place with BlackHat, will have an Oasis table. Need to start figuring out info we want to share
- Oasis will be at ROCA 
- Free pass for Linux Security Summit in Austin in June, reach out to Jane if you'd like to represent on behalf of Baseline

**Comment by GoldenBit0** [comment]
5/23/22:

- Mesh will have some members at Consensus, Jane will ship stickers to Sonal
- Identiverse charter with Oasis, we could supply with a Baseline slide. Mark will take a look to see if it's of interest. 
- Jack Leahy will let Jane know if anyone is interested in attending the Linux Security Summit on behalf of Baseline by Friday
- BlackHat attendance, $200 discount for new attendees, awaiting proposal approval.
- Provide is sponsoring DevConnect.
- All Things Open marketing barter again, will have more details in the future.
- Sonal will help with the Gartner event
- LendIt FInTech added to the tracker to track Boris' grant

**Comment by GoldenBit0** [comment]
6/6/22:

- Sonal to follow up with Jane on stickers
- Sonal, Keith, Mark attending Consensus, reviewing agenda to outline speakers/companies/events aligned with Baseline
- Mark to find connection with Identiverse <> Baseline for event*
- EEA is attending DevCon*, will keep in mind for Baseline related messaging. Sonal will follow up with Otto on attending DevCon & speaking about/demoing Baseline
- Sonal will follow up with Jane on Gartner event*
- Samrat will stay updated with Eth Online & Eth Singapore
- Nascom will be taking place again this year, and we have an early sponsorship opportunity. Sponsorship can include a headlining name, round tables, etc. 
- Crypto Expo Asia taking place in 2 weeks, Samrat will analyze agenda

**Comment by GoldenBit0** [comment]
[2022 Gartner Conferences](https://www.gartner.com/en/conferences/calendar)

**[Tech Growth & Innovation Conference](https://www.gartner.com/en/conferences/na/tech-growth-us)**
Details: July 12-13, Virtual, $1,450
Notes: Probably too short notice

**[Data & Analytics Summit](https://www.gartner.com/en/conferences/na/data-analytics-us)**
Details: August 22-24, Orlando FL, Standard price: $3,975

**[Identity & Access Management Summit](https://www.gartner.com/en/conferences/na/identity-access-management-us)**
Details: August 22-23, Las Vegas NV, $3,400

**[Security & Risk Management Summit](https://www.gartner.com/en/conferences/emea/security-risk-management-uk)**
Details: September 12-14, London UK, early bird price (exp. July 15) €3,000, €3,550

**[Data & Analytics Summit](https://www.gartner.com/en/conferences/apac/data-analytics-india)**
Details: September 19-20, Mumbai IN, early bird price (exp. July 22) INR 68,000, INR 78,500

**[Data & Analytics Summit](https://www.gartner.com/en/conferences/apac/data-analytics-australia)**
Details: November 7-8, Sydney AUS, early bird price (exp. September 9) AU$2,975, AU$3,475

**[IT Infrastructure, Operations & Cloud Strategies Conference](https://www.gartner.com/en/conferences/emea/infrastructure-operations-cloud-uk)**
Details: November 21-22, London UK, early bird price (exp. September 23) €2,625, €2,975

**[IT Infrastructure, Operations & Cloud Strategies Conference](https://www.gartner.com/en/conferences/na/infrastructure-operations-cloud-us)**
Details: December 6-8, Las Vegas NV, early bird price (exp. October 7) $3,625, $3,975

**Comment by GoldenBit0** [comment]
7/5/22:

Gartner Events
- There's a link for speaking opportunities, we can narrow it down to 3 events to apply* 
- Jane will message team to inquire for speaking slots* 

FinTech DevCon

- Marc needs deck completed by 7/21 for educational purposes, need to compile some decks
- Will add as agenda item to enablement workgroup meeting 7/7*

BlackHat
- Will have a booth that includes Baseline Protocol and stickers 
- Will have accessible QR codes for projects, need to decide where the code should go* 
- Sonal to send blurb to Carol and Jane for explaining baseline as it relates to crowd*

Notes
- Updating sticker design (possibly include QR code) (Sonal, Yoav, Mark, Jane, Keith)*
- Start looking at 2023 events

**Comment by GoldenBit0** [comment]
7/18/22:

FinTech DevCon 
- Marc using the enablement deck 
https://docs.google.com/presentation/d/1eTpOOsWARLAESG7aMco3VY42xVcufp146ZLh_Uh0Oxg/edit#slide=id.g12935f8f3cb_1_20
- Working group is connecting in next 2 days 

BlackHat
- Stickers will include QR code 
- Discount code for tickets expire 7/25

DevCon 
- Sonal & Keith sent in proposals for speaking, will update group 

ATO
- Will have a barter for presence again this year with open source booth
- Have a 101 event in March, call for presentations will open in the next months

BlackHat Europe
- In London in early Dec, submissions close this week, anyone in EU or wanting to go?

Notes
- Events for 2023 being pooled together, everyone check it out!

**Comment by GoldenBit0** [comment]
8/1/22:

BlackHat

- is next week, virtual and in person, going live on 8/10
- QR code displayed will lead to baseline site
- Action Item: outreach team will make minor UX updates prior to blackhat, for higher priority changes 

FinTech DevCon

- Marc is all set for event mid-way through month
- Action Item: Jane will check if there will be a virtual aspect 
- Action Item: Jane, Sonal, Marc, others will chat about business cards 
- Action Item: Team to start discussing post-outreach plan/follow ups

DevCon

- Speaker proposals not accepted

Nascom

- No slot as of now 
- Action Item: Samrat will follow up with team 

Gartner 

- Action Item: Sonal will reach out to Jane on 3 Gartner events by midweek
- Action Item: Sonal will add Gartner 2023 events to sheet

Eth Down Under
- https://ethdownunder.com/
- December 2022 in Sydney, AUS

Notes

- Action Item: Jane and Sonal to discuss new design for stickers with QR code

**Comment by GoldenBit0** [comment]
> **[IT Infrastructure, Operations & Cloud Strategies Conference](https://www.gartner.com/en/conferences/na/infrastructure-operations-cloud-us)**
> Details: December 6-8, Las Vegas NV, early bird price (exp. October 7) $3,625, $3,975

We should assess this event for 2022

**Comment by GoldenBit0** [comment]
[2023 Gartner Conferences](https://www.gartner.com/en/conferences/calendar)

**[Gartner Security & Risk Management Summit
](https://www.gartner.com/en/conferences/apac/security-risk-management-india)**
Details: Feb 13-14, Mumbai India

**[Gartner Identity & Access Management Summit](s/emea/identity-access-management-uk)**
Details: March 6-7, London, UK

**[Gartner Security & Risk Management Summit](https://www.gartner.com/en/conferences/apac/security-risk-management-australia)**
Details: March 28-29, Sydney, Australia

**[Gartner Security & Risk Management Summit](https://www.gartner.com/en/conferences/apac/security-risk-management-australia)**
Details: March 28-29, Sydney, Australia

**[Gartner Application Innovation & Business Solutions Summit
](https://www.gartner.com/en/conferences/na/applications-us)**
Details: May 22-24, Las Vegas, NV

**[Gartner Data & Analytics Summit
](https://www.gartner.com/en/conferences/emea/data-analytics-uk)**
Details: May 22-24, London, UK

**[Gartner Security & Risk Management Summit
](https://www.gartner.com/en/conferences/na/security-risk-management-us)**
Details: June 5-7, National Harbor, MD

**[Gartner Application Innovation & Business Solutions Summit](https://www.gartner.com/en/conferences/emea/applications-uk)**
Details: September 14-15, London, UK

**Comment by GoldenBit0** [comment]
8/15/22: 

BlackHat

- Jane will send a pic of booth for Oasis and baseline
- Carol is following up with visitors 

FinTech DevCon

- Request for developer focused presentation, Marc has explained scope of talk per submission
- Action Item: Jane finding out if there will be a virtual aspect

All Things Open
- Will have booth similar to BlackHat

EthVietnam 
- Samrat got invited, and proposed 2 workshops (ZK and baseline)
- Last week of Nov

EthKl
- Samrat presenting, will work with team to get deck put together 

EthDownUnder
- https://ethdownunder.com/

Nascom
- We will not have a presence this year

NATC

- Action Item: Samrat is following up

Question: Is IDC also doing conferences? With the upcoming reports?

Notes:
- Let me know if anyone might be interested in attending either of these events. I may be able to squeeze out a free pass.
OS Summit Latin America (virtual):
https://events.linuxfoundation.org/open-source-summit-latin-america/
OS Summit Europe (in-person/virtual)
https://events.linuxfoundation.org/open-source-summit-europe/

**Comment by GoldenBit0** [comment]
9/12/22:

Baseline presence being promoted: All Things Open and BlackHat. 

BlackHat Europe
- Oasis working on barter this for

EthVietnam
- Samrat is speaking 
- Asking for sponsorship (happening in November) to baseline community 

EthKL
- Samrat is speaking 

DevCon
- Mesh team is attending

Singapore Fintech Festival
- Samrat working on application

**Comment by GoldenBit0** [comment]
9/26/22:

DevCon

- Mesh and EEA teammates attending, Sonal reaching out to venues to find place for meet up, can contact places for 'after events' and won't be competing with agenda

All Things Open
- Oct 30-Nov 2 Marc H and team will be attending

Notes
- Team should shift through 2023 events list

**Comment by GoldenBit0** [comment]
10/24/22:

Devcon

- Teammates who were on ground will update in future in new community members, baseline show guests, and others.

All Things Open

- Marc. John and Oasis team on ground, will have a booth, new stickers
- Sonal will connect with organizer to possibly secure speaking slot for baseline

EthVietnam

- Samrat speaking and hosting a 'zk workshop', will be in touch with devs to plan

FinTech Signapore 
- Samrat may be attending

BlackHat Europe (London)
- Oasis will have a booth
- Any Baseline folks in London?

Notes
- Start 2023 event planning after next events/holidays

**Comment by GoldenBit0** [comment]
12/5/22:

All Things Open

- Group can go through speaker list to invite people to the Baseline Show (Samrat)

Notes

- Jane is updating list with events for 2023
- Sonal working on EthDenver speaker application
- Team is encouraged to start looking at events (COAs) for 2023

**Comment by GoldenBit0** [comment]
Opening a new issue for 2023.
